examples/test/test_resnet_vitis_ai_ptq_cpu.py

```diff
@@ -33,8 +33,8 @@ def setup():
 @pytest.mark.parametrize("system", ["local_system", "aml_system"])
 @pytest.mark.parametrize("olive_json", ["resnet_vitis_ai_ptq_cpu.json"])
 @pytest.mark.skipif(
-    version.parse(OrtVersion) >= version.parse("1.16.0"),
-    reason="VitisAIQuantization is not supported in ORT 1.16.0 with TensorsData",
+    version.parse(OrtVersion) == version.parse("1.16.0"),
+    reason="resnet is not supported in ORT 1.16.0 caused by https://github.com/microsoft/onnxruntime/issues/17627",
```

Comment:
maybe this issue?
https://github.com/microsoft/onnxruntime/issues/17619

Reply:
this line need to be reverted. It is related the TensorData changes

Reply:
thanks for catch